src/__tests__/starWarsIntrospection-test.ts

```diff
@@ -37,6 +37,7 @@ describe('Star Wars Introspection Tests', () => {
             { name: 'Droid' },
             { name: 'Query' },
             { name: 'Boolean' },
+            { name: 'ErrorBehavior' },
```

Comment:
I am unsure whether we should expose this as a `schema` property, this is in fact an internal property that tells us just how a GraphQL API will behave. Most GraphQL code I've seen treats it as such where when walking the introspection there is an explicit check for properties starting with `__` to be skipped.